recipes/fast_double_parser/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
should be `conan_basic_setup(TARGETS)`

Reply:
Thank you for reviewing! I will fix them.
This MR has been merged, should I open another MR? or is it okay to overwrite push?

Reply:
you have to open another PR

Reply:
Thank you. I will send another PR.